Rollbar/NSJSONSerialization+Rollbar.m

```diff
@@ -66,6 +69,23 @@ + (NSDictionary *)safeDataFromJSONObject:(id)obj {
     return safeData;
 }
 
++ (unsigned long)measureJSONDataByteSize:(NSData*)jsonData {
+    
```

Comment:
Isn't `-[NSData length]` the exact same thing? i.e. this method can just be `return jsonData.length;`

Reply:
does it mean that ".length" will take into account proper encoding? because we do want to make sure that the total bytes count of the payload is calculated using UTF8...

Reply:
You are taking some raw bytes encoding them as UTF8 and then asking how many bytes it took to encode the string as UTF8 which is the same as the raw bytes. Unless it is possible to have a certain number of raw bytes that when interpreted as UTF8 increases the number of bytes, but that I believe is impossible.

If you had the JSON string instead of the NSData object then you would definitely need to use lengthOfBytesUsingEncoding to get the size rather than length called on the string.

Under the hood I am pretty sure lengthOfBytesUsingEncoding is just doing strlen on the interior char* inside the NSString which is the same thing as length of the bytes used to generate the string.

Reply:
i see. changing it to: return jsonData.length;